pkg/api/docs/definitions/correlations.go

```diff
@@ -0,0 +1,33 @@
+package definitions
```

Comment:
I believe these are being moved out to live with the models themselves, @papagian is working on that.

Reply:
right in #52643
let me know if you prefer to merge it as is and handle it there or changing it as a part of this work

Reply:
i can handle it in a follow up PR if that's ok with you as i have 3 other pending PRs adding more endpoints (and definitions) on top of this one. is it ok?